### PR TITLE
add task2 & update task1

### PR DESCRIPTION
**Pull request by Leen-odeh3**
plz review ..

### CODE REVIEW

File: src/app/form/form.component.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!-- Bootstrap Link -->
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
```
Comment (suhaibmaree) [inline src/app/form/form.component.html]: if you are planning to use css then there is no need for bootstrap, you can remove it !

File: src/app/success-alert/success-alert.component.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+<div class="alert alert-success" role="alert">
```
Comment (suhaibmaree) [inline src/app/success-alert/success-alert.component.html]: use your own CSS class, remove "alert-success"

File: src/app/warning-alert/warning-alert.component.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<h1> Assignment-1</h1>
+<div class="alert alert-warning" role="alert">
```
Comment (suhaibmaree) [inline src/app/warning-alert/warning-alert.component.html]: remove "alert-warning" use your own class

File: src/app/form/form.component.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.9/angular.min.js"></script>
+<div class="container">
+  <div class="row">
+    <div class="col-xs-12">
+      <h1> Assignment-2</h1>
+
+      <div class="form">
+
+        <p>Hello, Plz Enter Your Name:</p>
+        <p>UserName: <input type="text" ng-model="name" (input)="submit($event)"></p>
```
Comment (suhaibmaree) [inline src/app/form/form.component.html]: This is not an implementation for two-way data binding you need to use [(ngModel)] instead
Check this doc https://angular.io/guide/forms

File: src/app/form/form.component.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        <p ng-bind="name">welcome! {{name}}</p>
```
Comment (suhaibmaree) [inline src/app/form/form.component.html]: Do not use ng-bind, you should use the {{}} as implementation of databinding

File: src/app/form/form.component.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+        <button (click)="onclick()" class="btn" [disabled]="name === ''">Click </button>
```
Comment (suhaibmaree) [inline src/app/form/form.component.html]: After testing, the button is not disabled when the input is empty.

File: src/app/form/form.component.ts [hunk 7]
```diff
@@ -0,0 +1,18 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-form',
+  templateUrl: './form.component.html',
+  styleUrls: ['./form.component.css']
+})
+export class FormComponent {
+  name:String='';
```
Comment (suhaibmaree) [inline src/app/form/form.component.ts]: format the line as following `name: String = '';`

File: src/app/form/form.component.ts [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
+  onclick(){
+    this.name='';
```
Comment (suhaibmaree) [inline src/app/form/form.component.ts]: add space for formating `this.name = '';`

File: src/app/form/form.component.ts [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+  }
+
+  submit(event:Event){
+    this.name=(event.target as HTMLInputElement).value;
```
Comment (suhaibmaree) [inline src/app/form/form.component.ts]: Why you are using an event here? it should be only assigning an empty string for the name property!

**Review by suhaibmaree — CHANGES_REQUESTED** [review]
Please read more about data binding and how to use it in the form 
Refer to this documentation https://angular.io/guide/forms

File: src/app/form/form.component.html (hunk 1 above)
Comment (Leen-odeh12) [reply src/app/form/form.component.html:2]: done remove it

File: src/app/success-alert/success-alert.component.html (hunk 2 above)
Comment (Leen-odeh12) [reply src/app/success-alert/success-alert.component.html:2]: done

File: src/app/form/form.component.html (hunk 4 above)
Comment (Leen-odeh12) [reply src/app/form/form.component.html:14]: done, Thank you.

File: src/app/form/form.component.html (hunk 5 above)
Comment (Leen-odeh3) [reply src/app/form/form.component.html]: okeyy.

File: src/app/form/form.component.ts (hunk 7 above)
Comment (Leen-odeh12) [reply src/app/form/form.component.ts:9]: wow , thanks alot.

File: src/app/form/form.component.ts (hunk 9 above)
Comment (Leen-odeh12) [reply src/app/form/form.component.ts:16]: okkkey all is done

**Pull request merged**